cost-analyzer/values.yaml

```diff
@@ -628,7 +702,7 @@ kubecostModel:
   ## The name of the Secret containing a bucket config for Federated storage.
   ## The contents should be stored under a key named federated-store.yaml.
   ## Ref: https://www.ibm.com/docs/en/kubecost/self-hosted/2.x?topic=cluster-long-term-storage-configuration
-  # federatedStorageConfigSecret: federated-store
+  # federatedStorage.existingSecret: federated-store
```

Comment:
We should delete the following from the `values.yaml` file? These configs are no longer allowed to be used.

```yaml
kubecostModel:
  federatedStorageConfigSecret: ""
  federatedStorageConfig: ""
```